packages/worker/src/tests/TestConfiguration.ts

```diff
@@ -129,10 +160,19 @@ class TestConfiguration {
 
   // TENANCY
 
-  createTenant = async (): Promise<User> => {
+  createTenant = async ({
+    addTenantToGlobalDb,
+  }: {
+    addTenantToGlobalDb: boolean
+  }): Promise<User> => {
     // create user / new tenant
     const res = await this.api.users.createAdminUser()
 
+    // This needs to be added because it was disabled for bulk testing: // https://github.com/Budibase/budibase/issues/6134
+    if (addTenantToGlobalDb) {
```

Comment:
The `globalDb` part here is a little confusing, is this the `global-info` db or the `tenant_global-db`? 

We really should have called the `global-info` db something else, such as `static-db` or `platform-db` to signify there is only one.

Reply:
As mentioned [here](https://github.com/Budibase/budibase/pull/9419#discussion_r1095598847) this is not needed anymore! 🎉